src/certificates/layout/certificates-sidebar/messages.js

```diff
@@ -0,0 +1,54 @@
+import { defineMessages } from '@edx/frontend-platform/i18n';
+
+const messages = defineMessages({
+  about: {
+    id: 'course-authoring.certificates.sidebar.about.title',
+    defaultMessage: 'Working with certificates',
+  },
+  aboutDescription_1: {
+    id: 'course-authoring.certificates.sidebar.about.description-1',
+    defaultMessage: 'Specify a course title to use on the certificate if the course\'s official title is too long to be displayed well.',
+  },
+  aboutDescription_2: {
+    id: 'course-authoring.certificates.sidebar.about.description-2',
+    defaultMessage: 'For verified certificates, specify between one and four signatories and upload the associated images. To edit or delete a certificate before it is activated, hover over the top right corner of the form and select {strongText} or the delete icon.',
+  },
+  aboutDescription_2_strong: {
+    id: 'course-authoring.certificates.sidebar.about.description-2.strong',
+    defaultMessage: 'Edit',
+  },
+  aboutDescription_3: {
+    id: 'course-authoring.certificates.sidebar.about.description-3',
+    defaultMessage: 'To view a sample certificate, choose a course mode and select {strongText}.',
+  },
+  aboutDescription_3_strong: {
+    id: 'course-authoring.certificates.sidebar.about.description-3.strong',
+    defaultMessage: 'Preview certificate',
+  },
+  about_2: {
+    id: 'course-authoring.certificates.sidebar.about2.title',
+    defaultMessage: 'Issuing certificates to learners',
+  },
+  about_2_Description_1: {
```

Comment:
These message identifiers aren't very descriptive.  Can we try and come up with some less generic ones?  (It's bad enough that none of the messages have descriptions, but I'm not going to hold the PRs up on this... for now.)